src/components/NftRewards/NftRewardsSection.tsx

```diff
@@ -141,14 +140,13 @@ export function NftRewardsSection() {
               >
                 <NftTierCard
                   rewardTier={rewardTier}
-                  rewardTierUpperLimit={
-                    rewardTiers?.[idx + 1]?.contributionFloor
+                  quantitySelected={
+                    payMetadata?.tierIdsToMint.filter(
+                      id => id === rewardTier.id ?? -1,
```

Comment:
`id` wont be undefined, so if `rewardTier.id` is, then this should still work right?

```suggestion
                      id => id === rewardTier.id,
```

Reply:
Yep, you're right.